WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -473,6 +468,36 @@ class PostListViewModel @Inject constructor(
         localPostIdForPublishDialog = null
     }
 
+    private fun restorePost(post: PostModel) {
+        // We need network connection to restore a post
+        if (!checkNetworkConnection()) {
+            return
+        }
+
+        postsBeingTrashedOrRestored.add(LocalId(post.id))
```

Comment:
I'd consider creating a method for these two lines - similar to `removeLocalPostIdFromPostsBeingTrashedOrRestored`.

Reply:
Addressed as part of 244e84938e455a0f8033b7fae3413a0dd21d3762.